examples/with-typescript/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "dev": "concurrently \"tsc --watch\" \"next\""
```

Comment:
@OskarKlintrot You could rebase this into the other "cleanup" commit :1st_place_medal: